Box.V2/Managers/BoxFilesManager.cs

```diff
@@ -1113,6 +1113,36 @@ private int GetTimeDelay(HttpResponseHeaders headers)
 
             return response.ResponseObject;
         }
+
+		/// <summary>
+		/// Representations are digital assets stored in Box. We can request the following representations: PDF, Extracted Text, Thumbnail,
+		/// and Single Page depending on whether the file type is supported by passing in the corresponding x-rep-hints header. This will generate a 
+		/// representation with a template_url. We will then have to either replace the {+asset_path} with <page_number>.png for single page or empty string
+		/// for all other representation types.
+		/// </summary>
+		/// <param name="id">Id of the file (Required).</param>
+		/// <param name="RepresentationType">Enum value of representation requested or string of representation requested (Required).</param>
+		/// <param name="contentDispositionType"> Optional value set to "inline" or "attachment" 
+		/// <param name="contentDispositionFilename"> Optional value to define the downloaded representation's file name.</param>
+		/// <returns>A full file object containing the updated representations template_url and state is returned.</returns>
+		/// </summary>
+		public async Task<BoxRepresentationCollection<BoxRepresentation>> GetRepresentationsAsync(string id, string representation,
+            string contentDispositionType = null, string contentDispositionFilename = null)
```

Comment:
Why not a complex request type like 
public async Task<BoxPreflightCheck> PreflightCheck(BoxPreflightCheckRequest preflightCheckRequest)

Would like to hear your reason here